src/main/java/org/jabref/gui/JabRefGUI.java

```diff
@@ -273,11 +276,9 @@ private void openDatabases() {
         // This is because importToOpen might have been used, which adds to
         // loadedDatabases, but not to getBasePanelCount()
```

Comment:
Please also adjust this comment (`getBasePanelCount()` is changed below to `getLibraryTabs().size()`) -- or change the code -- or add a TODO that one needs to think about the condition.

Reply:
Removed the comment and the check completly, since I removed the unused reference to libraryTab completly.